### DISCUSSION TITLE
GD keybinding doesn't seem to add to history

### DISCUSSION THREAD

**Discussion by wmonk** [NvChad/NvChad#2363] [Q&A]
Using `gd` to go to definition doesn't seem to support ctrl+o and then ctrl+i back to the position of the definition. Not sure if an nvchad issue or an issue of the neovim lsp.

**Comment by siduck** [comment]
@wmonk if that works with a non nvchad config then this definitely is a nvchad issue, test it

**Comment by georgejean** [comment]
@wmonk
Try to add this in `custom/mappings.lua`:
```lua
M.disabled = {
  n = {
    -- your disabled mappings ...
    ["<tab>"] = "",
  },
}
```
because `<tab>` (or `<C-I>`) is already used by tabufline to switch between buffers. You can always use `<S-tab>` for this purpose.
